### DISCUSSION TITLE
Understanding Archive Size presented on Vorta VS Borg CLI

### DISCUSSION THREAD

**Discussion by televisi** [borgbase/vorta#1927] [FAQ]
Hi,

I'm trying to understand the size presented on vorta vs Borg CLI.

Vorta version: 0.9.1
Borg version: 1.2.7
Ran on: Mac OSX Monterey (12.7.3)
Borg server: cloud

I ran 4 x backup so far.
I also ran ```borg compact``` CLI command before getting below screenshot/information.

The following is what displayed on Vorta when I click on 'Recalculate' on each archive:
<img width="683" alt="image" src="https://github.com/borgbase/vorta/assets/4546680/884bbdfe-c197-4ae7-8835-20b7974c7343">

The following is what resulted when I ran ```borg info``` on all of the archives:

```
documents_compressed-2024-02-05_T_00:11:01 
------------------------------------------------------------------------------
                       Original size      Compressed size    Deduplicated size
This archive:              119.10 GB             65.60 GB              2.65 MB
All archives:              476.37 GB            262.39 GB             50.23 GB

                       Unique chunks         Total chunks
Chunk index:                   90684               673124


documents_compressed-2024-02-05_T_22:46:18 
------------------------------------------------------------------------------
                       Original size      Compressed size    Deduplicated size
This archive:              119.10 GB             65.60 GB             90.09 kB
All archives:              476.37 GB            262.39 GB             50.23 GB

                       Unique chunks         Total chunks
Chunk index:                   90684               673124


documents_compressed-2024-02-08_T_14:58:06 
------------------------------------------------------------------------------
                       Original size      Compressed size    Deduplicated size
This archive:              119.09 GB             65.60 GB             44.37 kB
All archives:              476.37 GB            262.39 GB             50.23 GB

                       Unique chunks         Total chunks
Chunk index:                   90684               673124


documents_compressed-2024-02-08_T_15:32:03 
------------------------------------------------------------------------------
                       Original size      Compressed size    Deduplicated size
This archive:              119.09 GB             65.60 GB             44.37 kB
All archives:              476.37 GB            262.39 GB             50.23 GB

                       Unique chunks         Total chunks
Chunk index:                   90684               673124
```

As you can see none of them are showing the same result with what presented on Vorta.
Did I miss anything here? I'm confused with the correct size of my archive now

Thank you

**Comment by m3nu** [comment]
The sizes in the screenshot and the CLI output are the same. The three latest archives are very small due to very little changed data. Vorta tries to show the number that's closest to what would be stored on disk. Borg will show additional details.

**Reply by televisi** [reply]
Apologies for being thick.
I noticed two different numbers between the CLI and the Vorta UI?

E.g. for "documents_compressed-2024-02-05_T_00:11:01"
in UI, it shows 2649.9KB (which also translated to +/- 2.6TB)

While in CLI, only shows 65.6GB?

```
Original size      Compressed size    Deduplicated size
This archive:              119.10 GB             65.60 GB              2.65 MB
```
Did I miss anything here?

Thanks

**Reply by televisi** [reply]
Oh the other thing, when I ran ```du -shc``` on the folder, it shows the TOTAL disk usage is 47GB too... I'm confused where this number come from on vorta OR ```borg info``` CLI

```
/home/documents_compressed > du -shc
 47G	.
 47G	total
```

**Reply by real-yfprojects** [reply]
> E.g. for "documents_compressed-2024-02-05_T_00:11:01" in UI, it shows 2649.9KB (which also translated to +/- 2.6TB)
> 
> While in CLI, only shows 65.6GB?

$$ 2649.9 KB = 2.6499 MB \approx 2.65 MB $$

As you can see Vorta shows the deduplicated size of each archive which roughly corresponds to the size on the disk. However note that this is not entirely true. To quote the borg docs:

> Please note that the deduplicated sizes of the individual archives do not add up to the deduplicated size of the repository (“all archives”), because the two are meaning different things:
> 
> This archive / deduplicated size = amount of data stored ONLY for this archive = unique chunks of this archive. All archives / deduplicated size = amount of data stored in the repo = all chunks in the repository.



> Oh the other thing, when I ran `du -shc` on the folder, it shows the TOTAL disk usage is 47GB too...

Borg outputs total repository size of 50.23 GB. Not sure why the actual size is smaller.

**Comment by televisi** [comment]
> 2649.9KB=2.6499MB≈2.65MB

@real-yfprojects 
Oh my! you're right! my calculation was way off!

> Borg outputs total repository size of 50.23 GB. Not sure why the actual size is smaller.

I only have 4 x archives for this repository; and looks like each archive is only a small amount; I wonder why the total disk space is 47GB?
<img width="669" alt="image" src="https://github.com/borgbase/vorta/assets/4546680/d261befa-f307-48b1-81ba-f0993b1de440">

Did I miss anything here?

Thank you
